packages/voice/package.json

```diff
@@ -70,27 +70,25 @@
 		"ws": "^8.18.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.24.6",
-		"@babel/preset-env": "^7.24.6",
-		"@babel/preset-typescript": "^7.24.6",
 		"@discordjs/api-extractor": "workspace:^",
+		"@discordjs/opus": "^0.9.0",
 		"@discordjs/scripts": "workspace:^",
 		"@favware/cliff-jumper": "^4.1.0",
-		"@types/jest": "^29.5.12",
-		"@types/node": "^16.18.105",
+		"@types/node": "18.19.45",
+		"@vitest/coverage-v8": "2.0.5",
 		"cross-env": "^7.0.3",
 		"esbuild-plugin-version-injector": "^1.2.1",
 		"eslint": "^8.57.0",
 		"eslint-config-neon": "^0.1.62",
 		"eslint-formatter-pretty": "^6.0.1",
-		"jest": "^29.7.0",
-		"jest-websocket-mock": "^2.5.0",
 		"mock-socket": "^9.3.1",
```

Comment:
This dependency and its associated file can be removed as well
 https://github.com/nyapat/discord.js/blob/16390efe6e4f2035aa0fe365c8d593366beca7ce/packages/voice/__mocks__/ws.js